pkg/transport/filtering/object_filtering_map.go

```diff
@@ -31,6 +33,7 @@ type cleanObjectSpecificsFunction func(object *unstructured.Unstructured)
 func NewObjectFilteringMap() *ObjectFilteringMap {
 	filteringMap := map[schema.GroupVersionKind]cleanObjectSpecificsFunction{
 		corev1.SchemeGroupVersion.WithKind("Service"): cleanServiceFields,
+		batchv1.SchemeGroupVersion.WithKind("Job"):    cleanJob,
```

Comment:
nit:  cleanJobFields or change also to  cleanService  .  BTW, why not cleanJobObject and cleanServiceObject  ?

Reply:
Made the Service one shorter too.